src/Hadolint/Rules.hs

```diff
@@ -310,6 +312,41 @@ hasNoMaintainer = instructionRule code severity message check
     check (Maintainer _) = False
     check _ = True
 
+
+hasMaintainerLabel :: Rule
+hasMaintainerLabel dockerfile = instructionRuleState code severity message check [] dockerfile
+  where
+    code = "DL3012"
```

Comment:
Have reused the DL3012 here since it makes sense from a description perspective. Not sure how to edit the wiki. 
If someone can let me know how to, I would be glad to take that on for providing an example.

Reply:
the wiki is public for anyone to edit

Reply:
Thanks @lorenzo  - will try editing it.